sync.py

```diff
@@ -0,0 +1,95 @@
+import json
+import requests
+import yaml
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+
+LOKI_GATEWAY_URL = 'http://loki-gateway.glueops-core-loki.svc.cluster.local'
```

Comment:
Can you feed this in as an environment variable?

Reply:
totally missed this comment - yes, I'll add that

Reply:
added - merging